site/docs/embed.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+menu: docs
+title: Embed
+permalink: /docs/embed.html
+---
+
+This Document assumes that you have a Vega-Lite spec as a JavaScript object in a variable called `vlSpec`. You can find a simple example in our [tutorial]({{site.baseurl}}/tutorials/getting_started.html#embed).
```

Comment:
Document -> document
spec -> specification?  (Not sure, but we should agree on a convention and be consistent
JS object -> JSON  (JS object can have function, JSON cannot.)

a simple example -> an example specification